docs/samediff/templates/variables.md

```diff
@@ -1,3 +1,214 @@
-# SameDiff variables
+---
+title: Types of variables in SameDiff
+short_title: Variables
+description: What types of variables are used in SameDiff, their properties and how to switch these types.
+category: SameDiff
+weight: 3
+---
+
+# Variables in `SameDiff`
+
+##What are variables
```

Comment:
Minor issue: Looks like we need a space after `##` for it to render properly in markdown... https://github.com/eclipse/deeplearning4j/blob/871d5897f810953160a10dfdaeb4d0913c16a5f1/docs/samediff/templates/variables.md